.github/workflows/ci.yml

```
@@ -0,0 +1,150 @@
+name: CI
+on:
```

Can we please add an empty line between elements on the same level, that do not belong to each other?

```suggestion

on:
```

---

fixed with d07f7d6 __ALL OCCURRENCES__

---

Thanks a lot!